src/blocktestcore/action.cpp

```diff
@@ -103,7 +103,7 @@ unsigned int Action::normalizeUInt(const std::string& str,bool justFetch) const
 {
     std::string tmp=normalize(str,justFetch);
     if(tmp.empty())
-        return NAN;
+        return 0;
```

Comment:
NAN is different from 0, this change doesn't create regressions?

Reply:
`NAN` is a `double`, this method is returning `unsigned int`.
The automatic conversion (at least on gcc) casts `NAN` to `0`, therefore I doubt there is a possible regression here.

You could consider to return `UINT_MAX` (`numeric_limits<unsigned int>::max()`) instead, and consider that an invalid value, but that would definitely be a change of behavior.

Reply:
> NAN is different from 0, this change doesn't create regressions?

No, because as explained @drdanz it is just a warning fix, the behaviour remains the same.

Reply:
I didn't noticed that was int, ok for me